jsonate/json_encoder.py

```diff
@@ -98,6 +99,21 @@ def map_queryset(obj):
         fields = jsonate_fields(obj.model)
         return obj.values(*[field.name for field in fields])
 
+# Managers are typically hidden fields, and must be specified via meta fields
+@register_typemap(Manager)
+def map_manager(obj):
+    qs = obj.get_queryset()
```

Comment:
I'm not 100% sure, but I _think_ you can just return the queryset and then it'll trigger the type map for that type. (rather than copying the implementation for queryset here)